public/manifest.json

```diff
@@ -1,9 +1,9 @@
 {
-  "short_name": "React App",
-  "name": "Create React App Sample",
+  "short_name": "Guessemon",
+  "name": "Guess a demon",
   "icons": [
     {
-      "src": "favicon.ico",
+      "src": "cacoicon.ico",
```

Comment:
We could remove old icon if we don't plan to use it.

Reply:
Yeah, sure.